docs/kqueen.rst

```diff
@@ -152,26 +152,235 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
+11. Click ``Deploy Cluster``.
+12. Select the defined GCE provisioner.
+13. Specify the cluster requirements.
+14. Click ``Submit``.
+15. To track the cluster status, navigate to the KQueen main dashboard.
+
+To provision a Kubernetes cluster using the Azure Kubernetes Service:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Log in to https://portal.azure.com.
+2. Create an Azure Active Directory Application as described in the official Microsoft `Documentation <https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-azure-active-directory-application>`_.
+3. Copy the Application ID, Application Secret, Tenant ID (Directory ID), and Subscription ID to use in step 8.
+4. Set the ``Owner`` role to your Application in the Subscription settings to enable the creation of Kubernetes clusters. For details, see the latest steps in the Microsoft official documentation described in step 2. Manually save the Application secret because you are not able to retrieve the secret later. You provide the secret value with the application ID to log in as the application. Store the key value where your application can retrieve it.
+5. Navigate to the ``Resource groups`` tab and create a resource group. Copy the  ‘Resource group name’ to use in step 8.
+6. From the to ``Resource groups`` -> your_group -> Access Control (IAM) tab, verify that the Application has the ``Owner`` role in Resource group.
+7. Log in to the KQueen web UI.
+8. From the ``Create provisioner`` tab, select the AKS engine and set the following:
+	1. Set the ``Client ID`` as Application ID from p.3.
+	2. Set the ``Resource group name`` as Resource group name from p.4
+	3. Set the ``Secret`` as Application Secret from p.3
+	4. Set the ``Subscription ID`` as  Subscription ID from p.3
+	5. Set the ``Tenant ID`` as Tenant(Directory) ID from p.3
+9. In the KQueen web UI, click ``Deploy Cluster``.
+10. Select the AKS provisioner.
+11. Specify the cluster requirements.
+12. Specify the public SSH key to connect to AKS VM’s. For ssh access into created VM’s, need to  assign the public IP address to the VM (follow example `guide <https://gist.github.com/naumvd95/576d6e48200597ca89b26de15e8d3675>`_). Then it is possible to ``ssh azureuser@<<public_ip>> -i .ssh/your_defined_id_rsa``
+13. Click ``Submit``.
+14. To track the cluster status, navigate to the KQueen main dashboard.
+
+
+.. note::
+
+   The Admin Console in the Azure portal is supported only in Internet Explorer and Microsoft Edge and may fail to operate in other browsers due to Microsoft `issues <https://microsoftintune.uservoice.com/forums/291681-ideas/suggestions/18602776-admin-console-support-on-mac-osx>`_.
+
+**Pay attention** that AKS created separate resource during kubernetes cluster creating (it used defined Resource Group as prefix ). It may affect your billing. For ex.:
```

Comment:
.. note::

   The AKS creates a separate resource during the creation of a Kubernetes cluster and uses the defined Resource Group as a prefix. This may affect your billing. For example:
    .. code-block:: text

       Your Resource Group : Kqueen
       Additional cluster-generated Resource Group: MC_Kqueen_44a37a65-1dff-4ef8-97ca-87fa3b8aee62_eastus

   For more information, see  https://github.com/Azure/AKS/issues/3 and https://docs.microsoft.com/en-us/azure/aks/faq#why-are-two-resource-groups-created-with-aks.

Reply:
done